tests/src/test/scala/spire/laws/LawTests.scala

```diff
@@ -33,21 +33,21 @@ class LawTests extends FunSuite with Discipline {
   }
 
   // Float and Double fail these tests
-  checkAll("Int",        RingLaws[Int].euclideanRing)
-  checkAll("Long",       RingLaws[Long].euclideanRing)
+  checkAll("Int",        RingLaws[Int].cRing)
+  checkAll("Long",       RingLaws[Long].cRing)
```

Comment:
This, along with `Int` (and `Byte` + `Short`) can all be `signedGCDRing`, right? Should bump up the coverage a bit.